handlers/whatsapp/whatsapp.go

```diff
@@ -63,6 +64,50 @@ type eventPayload struct {
 		Text      struct {
 			Body string `json:"body"`
 		} `json:"text"`
+		Audio *struct {
+			File     string `json:"file"`
+			ID       string `json:"id"`
+			Link     string `json:"link"`
+			MimeType string `json:"mime_type"`
+			Sha256   string `json:"sha256"`
```

Comment:
So now that the outer is optional as a pointer, you can add some validation on the inner parts. (ie `validate:"required" where appropriate) That'll keep surprising things from happening downstream.

Reply:
good point, I'll add.

Reply:
done, I think.